tests/prague/eip7692_eof_v1/eip3540_eof_v1/test_section_order.py

```diff
@@ -0,0 +1,220 @@
+"""
+Different variations of EOF sections displacement
+"""
+
+from enum import Enum
+from typing import List
+
+import pytest
+
+from ethereum_test_tools import EOFTestFiller
+from ethereum_test_tools import Opcodes as Op
+from ethereum_test_tools.eof.v1 import (
+    AutoSection,
+    Bytes,
+    Container,
+    EOFException,
+    Section,
+    SectionKind,
+)
+from ethereum_test_tools.eof.v1.constants import NON_RETURNING_SECTION
+
+from .spec import EOF_FORK_NAME
```

Comment:
```suggestion
from .. import EOF_FORK_NAME
```

On latest commit in main, this is where the `EOF_FORK_NAME` is actually defined.